pkg/vf/rosetta_arm64.go

```diff
@@ -2,22 +2,36 @@ package vf
 
 import (
 	"fmt"
+	"os"
 
 	"github.com/Code-Hex/vz/v3"
 	log "github.com/sirupsen/logrus"
 )
 
-func checkRosettaAvailability(install bool) error {
-	availability := vz.LinuxRosettaDirectoryShareAvailability()
+var (
+	checkRosettaDirectoryShareAvailability = vz.LinuxRosettaDirectoryShareAvailability
+	doInstallRosetta                       = vz.LinuxRosettaDirectoryShareInstallRosetta
+)
+
+func (dev *RosettaShare) checkRosettaAvailability() error {
+	availability := checkRosettaDirectoryShareAvailability()
 	switch availability {
 	case vz.LinuxRosettaAvailabilityNotSupported:
 		return fmt.Errorf("rosetta is not supported")
 	case vz.LinuxRosettaAvailabilityNotInstalled:
-		if !install {
+		if !dev.InstallRosetta {
 			return fmt.Errorf("rosetta is not installed")
 		}
 		log.Debugf("installing rosetta")
-		if err := vz.LinuxRosettaDirectoryShareInstallRosetta(); err != nil {
+		if err := doInstallRosetta(); err != nil {
+			if dev.IgnoreIfMissing {
+				os.Stdout.WriteString("Rosetta installation failed. Continuing without Rosetta.")
```

Comment:
I'd just use `log.Info("Rosetta installation failed ...")`. Logging of the error is fine to keep in debug logs (`log.Debugf`)

Reply:
Done 👍